packages/mobile/src/invite/EnterInviteCode.tsx

```diff
@@ -192,7 +186,7 @@ export class EnterInviteCode extends React.Component<Props, State> {
                       <TextButton
                         style={styles.bottomButton}
                         onPress={this.onPressSkip}
-                        disabled={isRedeemingInvite || !!account}
+                        disabled={isRedeemingInvite}
```

Comment:
@jeanregisser why were we disabling on `account` true here? Looks to me that the `skipInvite` saga can handle the case where an account has already been created.

Reply:
I think the `!!account` check was there to prevent incorrect state to proceed.
If for some reason you end up on this screen (though it looks impossible today) and you already have an account, you shouldn't be able to skip to the next screen. As it might cause other unwanted state changes.

But you're right, `skipInvite` is able to handle that case and uses the existing account.
So your change looks fine to me.

@jmrossy what's your take on this? Just want to be sure we don't miss something here.